src/location.rs

```diff
@@ -35,6 +35,14 @@ impl SrcLocation {
     pub fn is_multiple(&self) -> bool {
         !self.is_single()
     }
+
+    /// Returns whether the given name is part of this location
+    pub fn contains(&self, name: &XorName) -> bool {
```

Comment:
function name better to be `is_src`?

Reply:
The type itself is already called **Src**Location so repeating `src` seems redundant to me. Also `contains` seems more appropriate than `is`, because it's about whether the location *contains* the name, rather than the location *is* the name.